packages/flutter/lib/src/material/material_app.dart

```diff
@@ -138,7 +141,16 @@ class _MaterialAppState extends State<MaterialApp> implements BindingObserver {
 
     ThemeData theme = config.theme ?? new ThemeData.fallback();
     Widget result = new MediaQuery(
-      data: new MediaQueryData(size: _size),
+      data: new MediaQueryData(
```

Comment:
I think this aspect of the change is a non-controversial improvement over the status quo, and it might be worth putting this in as a separate patch so that you can get it out of your tree quicker than the rest.